examples/example.js

```diff
@@ -1,4 +1,4 @@
-const fastJson = require('.')
+const fastJson = require('..')
```

Comment:
this file is linked in the package.json to support runkit
```suggestion
const fastJson = require('fast-json-stringify')
```

Reply:
Then we have to create one for runkit

Reply:
@Eomm  I updated path in require call and path to example.js in the package.json.